app/modules/notifications/schemas.py

```diff
@@ -8,18 +8,23 @@
 from flask_marshmallow import base_fields
 
 from app.extensions import ExtraValidationSchema
-from .models import Notification
+from .models import Notification, NotificationType, NotificationChannel
+
+
+class NotificationChannelSchema(ExtraValidationSchema):
+    def __new__(cls, *args, **kwargs):
```

Comment:
Far better way to do it. Thanks. Will make a note of this

Reply:
Yeah .. it took a lot of trial and error 😄 and I'm still not sure if this is the best way

Reply:
Yeah I guess that accessing private members is a bit clunky but this is at least scalable which is a big improvement